contrib/PackageKit.spec.in

```diff
@@ -199,6 +200,8 @@ popd > /dev/null
 %{_datadir}/PackageKit/helpers/test_spawn/search-name.sh
 %{_libexecdir}/packagekitd
 %{_libexecdir}/packagekit-direct
+%{python3_sitelib}/dnf-plugins/refresh_packagekit.py
+%{python3_sitelib}/dnf-plugins/__pycache__/*
```

Comment:
I am not knowledgeable in the RPM world, but it seems odd to ship Python bytecode in a package, as that's *very* specific to a specific Python version - so, likely you can drop shipping the caches.

Reply:
Fedora ships the bytecode cache, though I *was* handling that incorrectly according to guidelines.

https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_source_files_and_bytecode_cache

Reply:
Very interesting! Debian handles this completely differently (compiling stuff on the client upon installation and taking care that the bytecode is removed when uninstalling).